__tests__/integration.test.js

```diff
@@ -111,6 +111,33 @@ describe("GET /api/articles", () => {
         });
       });
   });
+  test("200: if a topic query is passed in, should only return topics with that query", () => {
+    return request(app)
+      .get("/api/articles?mitch")
```

Comment:
The syntax isn't quite right here. What if we had multiple queries, like topic and author? This should be `/api/articles?topic=mitch`